man/systemd.network.xml

```diff
@@ -1216,7 +1216,8 @@
         <varlistentry>
           <term><varname>Gateway=</varname></term>
           <listitem>
-            <para>As in the <literal>[Network]</literal> section.</para>
+            <para>Takes the gateway address or special value <literal>DHCP</literal>. If
+            <literal>DHCP</literal>, then the gateway address provided by DHCP is used.</para>
```

Comment:
hmm, on the right-hand side of settings we usually use lowercase wording, and probably should do that here too, i.e. "dhcp" rather than "DHCP"